src/components/IconButton.js

```diff
@@ -1,13 +1,18 @@
 import React from 'react';
 
+const variants = {
+  dark: 'text-gray-600 hover:text-gray-400',
+  light: 'text-gray-400 hover:text-gray-600',
+  white: 'text-white hover:text-white',
+};
+
 function IconButton({ children, title, variant, onClick, className }) {
+  const cssVariant = variants[variant] ?? variants['light'];
```

Comment:
```suggestion
  const cssVariant = variants[variant] || variants['light'];
```

Reply:
I leave this as is because in this case is only important null or undefined